lib/NeuraDialect/Transforms/CanonicalizeLiveInPass.cpp

```diff
@@ -88,20 +88,20 @@ LogicalResult promoteLiveInValuesToBlockArgs(Region &region) {
   while (changed) {
     changed = false;
 
```

Comment:
I feel `current_block` is more suitable over `pred_block`. I just did get how exactly the propagation is done. Please add a concrete example in the comment "So we need to propagate..." to demonstrate your solution if possible. Thanks~!